Code/Mantid/docs/source/algorithms/MonitorEfficiencyCorUser-v1.rst

```diff
@@ -0,0 +1,59 @@
+.. algorithm::
+
+.. summary::
+
+.. alias::
+
+.. properties::
+
+Description
+-----------
+
+Normalizes the counts by monitor counts with additional efficiency correction.
+
+Corrects the monitor efficiency according to the formula contained in TOFTOF's Parameters file.
+
+The values of the monitor counts and incident energy are stored in the SampleLogs of the input workspace.
+
+To date this algorithm only supports the TOFTOF instrument.
+
+
+Restrictions
+###################################
+
+A formula named "formula\_eff" must be defined in the instrument
+parameters file.
+
+Usage
+-----
+
+**Example**
+
```

Comment:
Would it be possible to have another example in here, where you show exactly what sort of log values are required? You can use http://docs.mantidproject.org/nightly/algorithms/SetInstrumentParameter-v1.html and http://docs.mantidproject.org/nightly/algorithms/AddSampleLog-v1.html to create such an example, and maybe http://docs.mantidproject.org/nightly/algorithms/CreateWorkspace-v1.html or http://docs.mantidproject.org/nightly/algorithms/CreateSampleWorkspace-v1.html to create the input workspace